### DISCUSSION TITLE
5.7.0-preview0

### DISCUSSION THREAD

**Discussion by arnaudlh** [aztfmod/terraform-azurerm-caf#1557] [General]
This preview release allows testing upgrade from 5.5.x to latest ```azurerm``` and ```azuread```.
Use at your own risks and benefits. 

Fork it, break it, PR it. 
Would love your feedbacks on:
1. Upgrade experience from 5.5.x. please review [./UPGRADE.MD](https://github.com/aztfmod/terraform-azurerm-caf/blob/int-5.7.0/UPGRADE.md) and amend. 
2. Feature augmentation to align with new azurerm and azuread providers.

<hr /><em>This discussion was created from the release <a href='https://github.com/aztfmod/terraform-azurerm-caf/releases/tag/5.7.0-preview0'>5.7.0-preview0</a>.</em>

**Comment by piroinno** [comment]
Hey, When will the testing end?

Aby chance you can get current main tagged and deployed into terraform registry too
